lib/Builder.js

```diff
@@ -127,16 +127,19 @@ Builder.prototype.request = function (method, data) {
 
   var defer = Q.defer()
   var req = this._db[method](data, defer.makeNodeResolver())
+  var startedAt = Date.now()
 
   var retryHandler = this.getRetryHandler()
   var table = this._table
 
-  var processingStartedAt, byteLength
+  var processingStartedAt, byteLength, requestLatencyInMs
```

Comment:
naming nit: i think we would usually call it "requestLatencyMs" and skip the "in"?